frontend/src/index.jsx

```diff
@@ -139,7 +140,7 @@ const router = createBrowserRouter(
                 />
                 {/*/!* Default to BudgetAndFunding *!/*/}
                 {/*<Route exact path="" element={<Navigate to={"budget-and-funding"} />} />*/}
-                <Route path="budget-lines" element={<Agreement />} />
+                <Route path="/budget-lines" element={<BudgetLineItemList />} />
```

Comment:
😄 wonder if we should setup redirects to add the all blis filter params?

so if you go to `/budget-lines` it should route you to `/budget-lines?filter=all-budget-line-items`?